packages/unleash-docker/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:5.8
+
+COPY . .
+
+RUN npm install --production && \
+    mkdir migrations && cp -r node_modules/unleash-server/migrations/* migrations/.
```

Comment:
yes, we should probably ship a command for running the migration from the unleash-server module itself or something.. anyhow, this works for now.